src/python/pants/init/target_roots_calculator.py

```diff
@@ -127,23 +127,38 @@ def create(cls, options, session, symbol_table, build_root=None):
     # initialization paths.
     changed_options = options.for_scope('changed')
     changed_request = ChangedRequest.from_options(changed_options)
-
+    owned_files = options.for_global_scope().owner_of
     logger.debug('spec_roots are: %s', spec_roots)
     logger.debug('changed_request is: %s', changed_request)
     scm = get_scm()
     change_calculator = ChangeCalculator(session, symbol_table, scm) if scm else None
+    owner_calculator = OwnerCalculator(session, symbol_table) if owned_files else None
+    logger.debug('owner_files are: %s', owned_files)
+    targets_specified = [1 for spec
+                         in (changed_request.is_actionable(), owned_files, spec_roots)
+                         if spec]
```

Comment:
the first item in the tuple would be a bool vs a `spec` - and the result of the list comp here is not targets but a count, so the naming feels a bit odd to me.

it's largely cosmetic, but how about something like this for readability?

```
target_modifiers = sum(1 for item in (changed_request.is_actionable(), owned_files, spec_roots) if item)
if target_modifiers > 1:
  ...
```

wdyt?

Reply:
Yes, that works too. I used a list in case someone in the future needed to see which ones were actually being triggered, and that info would be lost info due to the aggregating of counting. Then again, perhaps I was overthinking and we should aim for readability 🌮